src/wavefront-obj-parser.js

```diff
@@ -2,47 +2,50 @@ module.exports = ParseWavefrontObj
 
 // Map .obj vertex info line names to our returned property names
 var vertexInfoNameMap = {v: 'vertex', vt: 'uv', vn: 'normal'}
-// The returned properties that we will populate
-var parsedProperties = ['normal', 'uv', 'vertex', 'normalIndex', 'uvIndex', 'vertexIndex']
+
 function ParseWavefrontObj (wavefrontString) {
+  'use strict'
```

Comment:
Does this `use strict` help with anything? Do we need it? I'm thinking not, but please do correct me if I'm wrong!